addons/payment_xendit/models/payment_transaction.py

```diff
@@ -27,13 +27,13 @@ def _get_specific_rendering_values(self, processing_values):
         :rtype: dict
         """
         res = super()._get_specific_rendering_values(processing_values)
-        if self.provider_code != 'xendit':
+        if self.provider_code != 'xendit' and self.payment_method != 'card':
```

Comment:
Your runbot is red :eyes: 
```suggestion
        if self.provider_code != 'xendit' and self.payment_method_code != 'card':
```